src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const block = document.createElement('div');
+  const titleMessage = document.createElement('h2');
+  const descr = document.createElement('p');
```

Comment:
```suggestion
  const description = document.createElement('p');
```
It is better to use full names